tests/certs_test.go

```diff
@@ -126,57 +128,52 @@ var _ = Describe("Certs", func() {
 			KeyPath:  fmt.Sprintf("%s/bar.com.key", certPath)},
 	}
 
-	cleanUpCerts := func(certNames []string) {
-		certsListing := string(listCerts().Wait().Out.Contents()[:])
+	cleanUpCerts := func(profile string, certNames []string) {
+		certsListing := string(listCerts(testData.Profile).Wait().Out.Contents()[:])
 		if !strings.Contains(certsListing, "No certs") {
-			removeCerts(certNames)
+			removeCerts(profile, certNames)
 		}
 	}
 
-	cleanUpDomains := func(domains []string) {
+	cleanUpDomains := func(profile string, domains []string) {
 		for _, domain := range domains {
-			removeDomain(domain, testApp.Name)
+			removeDomain(profile, domain, testApp.Name)
 		}
 	}
 
 	Context("with an app yet to be deployed", func() {
 		BeforeEach(func() {
+			testData = initTestData()
 			gitInit()
 			testApp = App{Name: getRandAppName()}
-			createApp(testApp.Name)
+			createApp(testData.Profile, testApp.Name)
 			domain = getRandDomain()
 			certName = strings.Replace(domain, ".", "-", -1)
 			certNames = []string{certName}
 		})
 
 		AfterEach(func() {
-			cleanUpCerts(certNames)
+			cleanUpCerts(testData.Profile, certNames)
 		})
 
 		It("can add, attach, list, and remove certs", func() {
-			addDomain(domain, testApp.Name)
-
-			addCert(certName, certs["wildcard"].CertPath, certs["wildcard"].KeyPath)
-
-			Eventually(certsInfo(certName)).Should(Say("No connected domains"))
-
-			attachCert(certName, domain)
-
-			Eventually(certsInfo(certName)).Should(Say(domain))
+			addDomain(testData.Profile, domain, testApp.Name)
+			addCert(testData.Profile, certName, certs["wildcard"].CertPath, certs["wildcard"].KeyPath)
+			Eventually(certsInfo(testData.Profile, certName)).Should(Say("No connected domains"))
+			attachCert(testData.Profile, certName, domain)
+			Eventually(certsInfo(testData.Profile, certName)).Should(Say(domain))
 		})
 	})
 
 	Context("with a deployed app", func() {
-		once := &sync.Once{}
 
 		BeforeEach(func() {
+			testData = initTestData()
 			// Set up the test app only once and assume the suite will clean up.
-			once.Do(func() {
-				os.Chdir(exampleRepo)
-				appName := getRandAppName()
-				createApp(appName)
-				testApp = deployApp(appName)
-			})
+			os.Chdir(exampleRepo)
```

Comment:
We can remove the comment above now that we are no longer only deploying once for the subsequent assertions